gems/pending/metadata/linux/LinuxSystemd.rb

```diff
@@ -32,7 +32,8 @@ def parse_systemd
     end
 
     def files(unit_extension)
-      (SYSTEM_DIRS + USER_DIRS).flat_map do |dir|
+      dirs = my_dirs
+      dirs.flat_map do |dir|
```

Comment:
I think I had actually accidentally deleted it in an earlier version of the WIP PR during a rebase and then put it back in.  That's my story and I"m sticking to it.  If you would like it removed let me know.